### PR TITLE
Added an extra command

### PR DESCRIPTION
**Pull request by nsisodiya**

### CODE REVIEW

File: docker-without-sudo.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Optionally change the username to match your preferred user.
 ```console
 $ sudo gpasswd -a $USER docker
 ```
+It is does not works, try this
```
Comment (papb) [inline docker-without-sudo.md:16]:
```suggestion

If is does not work, try this:
```

File: docker-without-sudo.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Optionally change the username to match your preferred user.
...
+
+```console
+sudo setfacl -m user:$USER:rw /var/run/docker.sock
+```
+replace `$USER` with your home username
```
Comment (papb) [inline docker-without-sudo.md:21]:
```suggestion

(replace `$USER` with the desired username)
```

**Comment by sindresorhus** [comment]
Please put a little bit more effort into this.

---

- https://github.com/sindresorhus/guides/pull/51
- https://github.com/sindresorhus/guides/pull/49

**Comment by nsisodiya** [comment]
Done

File: docker-without-sudo.md [hunk 3]
```diff
@@ -14,6 +14,14 @@ Optionally change the username to match your preferred user.
 $ sudo gpasswd -a $USER docker
 ```
 
+If is does not work, try this:
```
Comment (sindresorhus) [inline docker-without-sudo.md:17]: This should describe in what cases the above command might not work and what the below command does.

File: docker-without-sudo.md [hunk 4]
```diff
@@ -14,6 +14,14 @@ Optionally change the username to match your preferred user.
...
+
+```console
+sudo setfacl -m user:$USER:rw /var/run/docker.sock
```
Comment (sindresorhus) [inline docker-without-sudo.md:20]: Use the long-flag for `-m`

File: docker-without-sudo.md (hunk 3 above)
Comment (sindresorhus) [reply docker-without-sudo.md:17]: Also typo

File: docker-without-sudo.md [hunk 5]
```diff
@@ -14,6 +14,14 @@ Optionally change the username to match your preferred user.
...
+```
+
+(replace `$USER` with the desired username)
```
Comment (sindresorhus) [inline docker-without-sudo.md:23]: I don't think this is correct. `$USER` contains the username.

**Comment by sindresorhus** [comment]
Give the PR a proper descriptive title.

**Pull request closed without merging**